1.9/api/index.md

```diff
@@ -0,0 +1,38 @@
+---
+post_title: DC/OS API Reference
+nav_title: API Reference
+menu_order: 4
+---
+
+The DC/OS API is a collection of routes backed by [DC/OS Components](/docs/1.9/overview/architecture/components/) made available through [Admin Router](/docs/1.9/overview/architecture/components/#admin-router), an API gateway.
```

Comment:
s/s `The DC/OS API is a collection of routes backed by [DC/OS components](/docs/1.9/overview/architecture/components/) that are made available through an API gateway called the [Admin Router](/docs/1.9/overview/architecture/components/#admin-router).`